src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/RevurderFraService.kt

```diff
@@ -23,7 +23,7 @@ class RevurderFraService(
         val behandling = behandlingRepository.findByIdOrThrow(behandlingId)
 
         feilHvis(behandling.status.behandlingErLåstForVidereRedigering()) {
-            "Kan ikke oppdatere revurder fra når behandlingen er låst"
+            "Kan ikke oppdatere revurder fra når behandlingen har status ${behandling.status.visningsnavn()}."
```

Comment:
Vi har jo mange feilmeldinger som lyder _"Kan ikke oppdatere x fordi behandlingen er i status y"_

Forslag: Skulle vi like godt samlet disse under én? Da blir det tydeligere at det ikke bare en enkeltfelter i behandlingen som ikke lar seg oppdatere, men at faktisk hele behandlingen er låst. 

Går da an å lage en fellesfunksjon for dette, plassert f.eks. i `BehandlingStatus`, noe ala 

```kotlin
    fun kastFeilHvisBehandlingenErLåst() {
        feilHvis(
            behandlingErLåstForVidereRedigering(),
        ) { "Kan ikke gjøre endringer på denne behandlingen fordi den har status ${visningsnavn()}." }
    }
```